### PR TITLE
[openvpn] Add pmacct json-log audit support

### PR DESCRIPTION
**Pull request by trublast**
## Description
Add `pmacct` JSON-audit-log support for OpenVPN.

## Why do we need it, and what problem does it solve?
Sometimes it's required to audit user activity over VPN.

## What is the expected result?
Add a sidecar with `pmacctd`.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: openvpn
type: feature
summary: Added `pmacct` JSON-log audit support for OpenVPN.
```

### CODE REVIEW

**Review by z9r5 — APPROVED** [review]

**Review by vitaliy-sn — CHANGES_REQUESTED** [review]
Need to fix tests and DCO

**Comment by z9r5** [comment]
@trublast Please, rebase to the fresh main branch to pass linters.

**Review by vitaliy-sn — APPROVED** [review]
lgfm

File: modules/500-openvpn/docs/README.md [hunk 1]
```diff
@@ -26,3 +26,9 @@ The following parameters are pushed to the user's computer after connecting to t
 - a route to the local network;
 - a route to the cluster service network;
 - a route to the Pod network.
+
+## User's traffic audit
+
+The module can log VPN connections in JSON format. Traffic is grouped
+by `src_ip, dst_ip, src_port, dst_port, ip_proto` fields. Container logs can be collected and sent for storage or analysis
+using the [log-shipper](../460-log-shipper/) module.
```
Comment (apolovov) [inline modules/500-openvpn/docs/README.md:34]:
```suggestion
The module can log user's activity via VPN in JSON format. Traffic is grouped
by `src_ip, dst_ip, src_port, dst_port, ip_proto` fields. Container logs can be collected and sent to storage for further audit
using the [log-shipper](../460-log-shipper/) module.
```

File: modules/500-openvpn/docs/README_RU.md [hunk 2]
```diff
@@ -26,3 +26,7 @@ Web-интерфейс проинтегрирован с модулем [user-au
 - маршрут в локальную сеть;
 - маршрут в сервисную сеть кластера;
 - маршрут в сеть Pod'ов.
+
+## Аудит пользовательских соединений
+
+Модуль позволяет включить логирование VPN-подключений в JSON-формате. Группировка трафика происходит по полям `src_ip, dst_ip, src_port, dst_port, ip_proto`. Логи контейнера могут быть собраны и отправлены на хранение или анализ с помощью модуля [log-shipper](../460-log-shipper/).
```
Comment (apolovov) [inline modules/500-openvpn/docs/README_RU.md:32]:
```suggestion
Модуль позволяет включить логирование пользовательской активности через VPN в JSON-формате. Группировка трафика происходит по полям `src_ip, dst_ip, src_port, dst_port, ip_proto`. Логи контейнера могут быть собраны и отправлены на хранение для последующего аудита с помощью модуля [log-shipper](../460-log-shipper/).
```

File: modules/500-openvpn/openapi/config-values.yaml [hunk 3]
```diff
@@ -232,6 +232,15 @@ properties:
     type: boolean
     default: false
     x-examples: [true, false]
+  pmacctEnabled:
+    description: |
+      Enable logging of VPN connections in JSON format.
+
+      All connections within `tun` interfaces will be collected via `libpcap` and logged
+      as `{"event_type": "purge", "ip_src": "172.25.175.10", "ip_dst": "10.222.0.10", "port_src": 32172, "port_dst": 53, "ip_proto": "udp", "packets": 1, "bytes": 53}`
```
Comment (apolovov) [inline modules/500-openvpn/openapi/config-values.yaml:240]:
```suggestion
      Enable logging of user's activity via VPN in JSON format.

      All connections within `tun` interfaces will be collected via `libpcap` and logged
      as `{"event_type": "purge", "ip_src": "172.25.175.10", "ip_dst": "10.222.0.10", "port_src": 32172, "port_dst": 53, "ip_proto": "udp", "packets": 1, "bytes": 53}`
```

File: modules/500-openvpn/openapi/doc-ru-config-values.yaml [hunk 4]
```diff
@@ -136,6 +136,15 @@ properties:
       Структура, аналогичная  `spec.tolerations` в Kubernetes Pod.
 
       Если ничего не указано или указано `false` — будет [использоваться автоматика](https://deckhouse.ru/documentation/v1/#выделение-узлов-под-определенный-вид-нагрузки).
+  pmacctEnabled:
+    description: |
+      Включить логирование VPN-подключений в JSON-формате.
+      
+      Все соединения внутри `tun` интерфейсов будут собраны через `libpcap` и записаны в лог
+      в виде `{"event_type": "purge", "ip_src": "172.25.175.10", "ip_dst": "10.222.0.10", "port_src": 32172, "port_dst": 53, "ip_proto": "udp", "packets": 1, "bytes": 53}`.
```
Comment (apolovov) [inline modules/500-openvpn/openapi/doc-ru-config-values.yaml:144]:
```suggestion
      Включить логирование пользовательской активности через VPN в JSON-формате.
      
      Все соединения внутри `tun` интерфейсов будут собраны через `libpcap` и записаны в лог
      в виде `{"event_type": "purge", "ip_src": "172.25.175.10", "ip_dst": "10.222.0.10", "port_src": 32172, "port_dst": 53, "ip_proto": "udp", "packets": 1, "bytes": 53}`.
```

**Review by apolovov — APPROVED** [review]

**Review by apolovov — APPROVED** [review]

**Pull request merged**